src/Views/Divers/Details.cshtml

```diff
@@ -175,9 +207,74 @@
 
 @section scripts {
     <script type="text/javascript">
+        var secondaryColar = "#CED4DA";
+        var dangerColar = "#DC3545";
+        var year = document.getElementById("year");
+        var minutes = document.getElementById("minutes");
+        var yearLabel = document.getElementById("yearLabel");
+        var minutesLabel = document.getElementById("minutesLabel");
         var model = @Json.Serialize(Model);
+        var workingTime = model.workingTime;
+
+        function checkYear()
```

Comment:
you need to separate the change of styles and check of data into different functions